### PR TITLE
Removing unused code from gatsby-node file

### PR DESCRIPTION
**Pull request by iFlameing**

### CODE REVIEW

File: src/gatsby-node.js [hunk 1]
```diff
@@ -13,8 +12,6 @@ import {
 } from './utils';
 
 const ComponentNodeTypes = new Set(['PloneBreadcrumbs', 'PloneNavigation']);
-const DownloadableContentTypes = new Set(['Image', 'File']);
-
 // TODO: Make DownloadableContentTypes configurable
```
Comment (datakurre) [inline src/gatsby-node.js]: Probably also this comment should be removed.

Meanwhile I should go to git blame to see why I did add that...

Comment (iFlameing) [reply src/gatsby-node.js]: ok

Comment (datakurre) [reply src/gatsby-node.js]: @iFlameing That comment is still relevant, but it should be moved to utils.js next to the DownloadableContentTypes definition.

Comment (iFlameing) [reply src/gatsby-node.js]: ok! I will make this change.

**Pull request merged**